dev/archery/archery/crossbow/core.py

```diff
@@ -665,7 +665,7 @@ def get_version(root, **kwargs):
     if 'dev' not in tag:
         major += 1
 
-    return "{}.{}.{}.dev{}".format(major, minor, patch, version.distance)
+    return "{}.{}.{}.dev{}".format(major, minor, patch, version.distance or 0)
```

Comment:
Probably due to the release branch: the latest tag on the release branch is not reachable from the PR's HEAD commit.